src/app/shared/components/dev-playground/dev-playground.component.html

```diff
@@ -7,3 +7,24 @@
 <ng-container *ngIf="(content$ | async | unwrapStatus) as res">
   {{ res | json }}
 </ng-container>
+
+<!-- TODO: transaction examples. clean when it will be used elsewhere. -->
+<!--    <button app-action-button-->
+<!--            class="bg-blue-500 disabled:bg-gray-400 text-white p-2 rounded-md mt-2"-->
+<!--            text="Send USDC"-->
+<!--            [onClick]="sendUSDC('0x338c7f21E0195449',-->
+<!--            '0.000000000000000002').bind(this)">-->
+<!--    </button>-->
+
+<!--    <button app-action-button-->
+<!--            class="bg-blue-500 disabled:bg-gray-400 text-white p-2 rounded-md mt-2"-->
+<!--            text="Send MATIC"-->
+<!--            [onClick]="sendMATIC('0x7FEB921abdd0d05100', '0.02').bind(this)">-->
+<!--    </button>-->
+
+<button app-action-button
+        class="bg-blue-500 disabled:bg-gray-400 text-white p-2 rounded-md mt-2"
+        text="Sign Message"
+        loadingText="Signing message..."
+        [onClick]="signMessage.bind(this)">
+</button>
```

Comment:
We'll need those things in the future, so I moved it to dev playground component.